CurrencyConverter/CurrencyConverter/ViewController.swift

```diff
@@ -8,6 +8,12 @@
 
 import UIKit
 
+enum Currency {
+    case cad
+    case mxn
+}
+var currency: Currency?
```

Comment:
When I start the app the default currency button and labels are set to CAD, but what I get is the conversion to MXN. This can be fixed by changing this line to:

```
var currency: Currency = .cad
```